### DISCUSSION TITLE
Set scope using the "all in one (recommended)" option in oauth

### DISCUSSION THREAD

**Discussion by mtkgeek** [pocketbase/pocketbase#5177] [Q&A]
There are two ways of integrating oauth. The all in one automatic option provides the best experience and is recommended but is there a way to set scopes without using the other manual method? A simple argument to the method would have been nice.

**Comment by ganigeorgiev** [comment]
You can set the `scopes` [OAuth2AuthConfig option](https://github.com/pocketbase/js-sdk/blob/ab68644b4ba586afd6c7659cad5eba9b491bb908/src/services/RecordService.ts#L60-L75):

```js
await pb.collection("example").authWithOAuth2({
    provider: "google",
    scopes: ["scope1", "scope2"]
})
```
